### ISSUE TITLE
Error in engine.py, using python3.9 (posixpath.py) when template_path is None

### ISSUE THREAD

**Issue by Cristianasp** [mikeckennedy/fastapi-chameleon#6]
Hello, got this error trying to run using ch5-viewmodels from TalkPython Course... template_path is None.

I believe the problem is that `fastapi_chameleon.global_init()` is called after `from fastapi_chameleon import template` in home.py

```
Exception has occurred: TypeError       (note: full exception trace is shown but execution is paused at: _run_module_as_main)
expected str, bytes or os.PathLike object, not NoneType
  File "/home/cristiana/anaconda3/envs/fastapi/lib/python3.9/posixpath.py", line 76, in join
    a = os.fspath(a)
  File "/home/cristiana/anaconda3/envs/fastapi/lib/python3.9/site-packages/fastapi_chameleon/engine.py", line 72, in response_inner
    if not os.path.exists(os.path.join(template_path, template_file)):
  File "/home/cristiana/AAA-wip/Cursos-Programming/Curso-TalkPython-FastAPI-FullWeb/web-applications-with-fastapi-cris/views/home.py", line 13, in <module>
    def index(request: Request):
  File "/home/cristiana/AAA-wip/Cursos-Programming/Curso-TalkPython-FastAPI-FullWeb/web-applications-with-fastapi-cris/main.py", line 7, in <module>
    from views import home
  File "/home/cristiana/anaconda3/envs/fastapi/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/cristiana/anaconda3/envs/fastapi/lib/python3.9/runpy.py", line 97, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "/home/cristiana/anaconda3/envs/fastapi/lib/python3.9/runpy.py", line 268, in run_path
    return _run_module_code(code, init_globals, run_name,
  File "/home/cristiana/anaconda3/envs/fastapi/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/cristiana/anaconda3/envs/fastapi/lib/python3.9/runpy.py", line 197, in _run_module_as_main (Current frame)
    return _run_code(code, main_globals, None,
```

**Comment by Cristianasp** [comment]
Sorry, I will open this issue in TalkPython Course repo.

**Comment by mikeckennedy** [comment]
No problem. We already fixed it here on #7
